Data/Sys/GameSettings/010E01.ini

```diff
@@ -0,0 +1,22 @@
+# 010E01 - Wii Backup Disc v1.31
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId =
```

Comment:
OK. I think we should be consistent, so do the rest of the INIs typically use nothing or 0?